app/components/application_component.rb

```diff
@@ -1,37 +1,22 @@
 # frozen_string_literal: true
 
+# ApplicationComponent provides the base class for all ViewComponents
 class ApplicationComponent < ViewComponent::Base
-  DEFAULT_INITIAL_UNIT = "imperial"
-
-  def raise_if_invalid_value!(attribute, value, options = {})
-    return if options.include?(value)
+  include ApplicationComponentHelper
 
-    raise ArgumentError, "Invalid #{attribute}: #{value}. Must be one of: #{options.join(", ")}"
-  end
-
-  private
+  DEFAULT_INITIAL_UNIT = "imperial"
 
+  # Make unit class methods public for backwards compatibility
```

Comment:
Remove this comment and these methods (`unit_class_metric` and `unit_class_imperial`)